sky/engine/public/platform/sky_settings.h

```diff
@@ -17,6 +17,7 @@ struct SkySettings {
   bool start_paused = false;
   bool enable_dart_checked_mode = false;
   bool trace_startup = false;
+  bool complete_timeline = false;
```

Comment:
That makes sense.  At this layer of the system, we'll probably need a different name given that we'll still use a ring buffer for the `base` trace events.